awx/ui/client/src/login/authenticationServices/pendo.service.js

```diff
@@ -43,13 +43,13 @@ export default
 
             setDetailed: function(options, config) {
                 // Detailed mode
-                    // VisitorId: username+hash of license_key
+                    // VisitorId: userid+hash of license_key
                     // AccountId: hash of license_key from license
-                    // email: contact_email from license OR email from user account
+                    // email: <blank>
 
-                options.visitor.id = $rootScope.current_user.username + '@' + config.deployment_id;
+                options.visitor.id = $rootScope.current_user.id + '@' + config.deployment_id;
                 options.account.id = config.deployment_id;
-                options.visitor.email = $rootScope.current_user.email;
+                options.visitor.email = "";
```

Comment:
Can we just remove email? I checked the Pendo docs and `email` is an optional setting. 

FWIW, the only required key/value pair for `visitor` is the `id`.

Reply:
Maybe? I was thinking we would overwrite the existing one that's stored.... but then that wouldn't matter if we're changing 'id' at the same time.